prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <h1 style="color:blue">My favorite Start Wars character is Ashoka!</h1>
```

Comment:
Hi Daniel, be sure to put all visible content, like h1 elements within the `<body></body>` element of the document.